doc/Command-Reference.md

```diff
@@ -712,7 +712,25 @@ This command displays the cause of the previous reboot
   admin@sonic:~$ show reboot-cause
   User issued reboot command [User: admin, Time: Mon Mar 25 01:02:03 UTC 2019]
   ```
+#### Applicable only to SmartSwitch platforms
+**show reboot-cause all**
 
+This command displays the cause of the previous reboot for the Switch and the enabled DPUs
+
+- Usage:
+  ```
+  show reboot-cause all
+  ```
+
+- Example:
+  ```
+  root@MtFuji:~$  show reboot-cause all
+  Device    Name                 Cause                      Time    User
+  --------  -------------------  ----------                 ------  ------
+  SWITCH    2024_07_24_20_43_22  Power Loss                 N/A     N/A
+  DPU2      2024_07_24_20_43_22  Software causes (Reboot)   N/A     N/A
+  DPU1      2024_07_24_20_43_22  Software causes (Reboot)   N/A     N/A
+  ```
```

Comment:
@rameshraghupathy What will be the output for DPU that is in admin shut?

Reply:
There won't be any output for the DPUs that are in admin DOWN state.
@prgeor

Reply:
@rameshraghupathy can you mention that in the CLI doc?